GsaGH/Helpers/Export/Loads/Lists.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using Grasshopper.Kernel;
+using Grasshopper.Kernel.Types;
+using GsaAPI;
+using GsaGH.Helpers.GH;
+using GsaGH.Parameters;
+using OasysUnits;
+using OasysUnits.Units;
+using Rhino.Geometry;
+
+namespace GsaGH.Helpers.Export {
+  internal partial class Loads {
+
+    internal static List<GsaList> GetLoadLists(List<GsaLoad> loads) {
+      var loadLists = new List<GsaList>();
+      foreach (GsaLoad load in loads.Where(gsaLoad => gsaLoad != null)) {
+        GsaList list = GetLoadLists(load);
+        if (list != null) {
+          loadLists.Add(list);
+        }
+      }
+      return loadLists;
+    }
+    private static GsaList GetLoadLists(GsaLoad load) {
+      switch (load.LoadType) {
```

Comment:
shouldn't we first check if load is not null?

Reply:
https://github.com/arup-group/GSA-Grasshopper/pull/414/commits/c8e670a3faa98d90c5389c9a8ea5f6c078e7d5ae